<launch>
    <arg name = "destination" default = "172.16.1.1" />
    <arg name = "port_send" default = "6001" />
    <arg name = "port_receive" default = "6000" /> 

    <node pkg = "mabx_connector" type = "mabx_sender" name = "mabx_sender">
        <param name = "destination" value = "$(arg destination)" />
        <param name = "port_send" value = "$(arg port_send)" />
    </node>

    <node pkg = "mabx_connector" type = "mabx_receiver" name = "mabx_receiver">
        <param name = "port_receive" value = "$(arg port_receive)" />
    </node>
</launch>